test/resources/case20_delete_objects/case20_change_config_policy_not_prune.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: case20-2-name-changed
+  namespace: managed
+spec:
+  remediationAction: enforce
+  pruneObjectBehavior: DeleteAll
```

Comment:
Could `pruneObjectBehavior` be set to `None` or removed in this YAML and then remove the test code that patches the value now that this is a separate policy?

Reply:
good Thanks! I missed

Reply:
My purpose which created this yaml was separate the spec from "Clean up old object when configurationpolicy is changed" spec Thanks!!